src/main/resources/view/MainWindow.fxml

```diff
@@ -14,6 +14,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
+<?import javafx.scene.control.Label?>
```

Comment:
Is this perhaps a wrong import line?

Reply:
Yes, I've deleted this part. Thanks for pointing out 👍